### DISCUSSION TITLE
load plugin报错

### DISCUSSION THREAD

**Discussion by wzg-zhuo** [TuGraph-family/tugraph-db#163] [Q&A]
(400, 'Failed to compile plugin. \nStdout:----\n\nStderr:----\n/var/lib/lgraph/data/5FFB2BF9573C8B03590AEB4ACCDD3F17/_cpp_plugin_/_fma_bfs1540383426_1675404216089603349.tmp/_fma_bfs.cpp:16:10: fatal error: ./algo.h: No such file or directory\n #include "./algo.h"\n          ^~~~~~~~~~\ncompilation terminated.\n')

**Comment by knightast** [comment]
这个看报错，就是没有algo.h，改到同一个文件里，或者整一个zip包放上去
